src/PrestaShopBundle/Form/Exception/InvalidConfigurationDataErrorCollection.php

```diff
@@ -30,6 +30,7 @@
 
 use PrestaShop\PrestaShop\Core\Data\AbstractTypedCollection;
 
+/** @deprecated and will be removed in 9.0 */
 class InvalidConfigurationDataErrorCollection extends AbstractTypedCollection
```

Comment:
You can call @trigger_error before the class to make sure in will appear in the logs.